packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -368,6 +424,8 @@ class FlutterErrorBuilder {
     _parts.add(node.toDiagnosticsNode(name: name, style: DiagnosticsTreeStyle.shallow));
   }
 
+  /// Adds an property with an `Iterable<T>` [value] that can be displayed with
```

Comment:
Probably all or none of the methods should mention that the method can take a custom style to tweak the visual display.

Reply:
Okay, I simplified it to "Adds a property with an `Iterable<T>` [value] to the error report." for now.